src/services/article.py

```
@@ -250,17 +212,9 @@ def delete_article_by_author(self, id: int, current_user: User) -> None:
     def delete_article_by_executive(self, id: int, current_user: User) -> None:
```

Ruff (0.15.12)

[error] Function argument `id` is shadowing a Python builtin

(A002)